README.md

```diff
@@ -107,6 +107,8 @@ $ cmake -G Ninja .. \
     -DCMAKE_BUILD_TYPE=DEBUG
 $ ninja
 $ ninja check-circt
+$ ninja check-circt-integration # Run the integration tests, which require
```

Comment:
I'd drop "which require additional software".  The tests are properly feature gated.